src/plugins/unified_search/public/query_string_input/add_filter_popover.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useState } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiFlexItem,
+  EuiButtonIcon,
+  EuiPopover,
+  EuiButtonIconProps,
+  EuiToolTip,
+} from '@elastic/eui';
+import { Filter } from '@kbn/es-query';
+import type { DataView } from '../../../data_views/public';
+import { FilterEditorWrapper } from './filter_editor_wrapper';
+
+interface AddFilterPopoverProps {
+  indexPatterns?: Array<DataView | string>;
+  filters: Filter[];
+  timeRangeForSuggestionsOverride?: boolean;
+  onFiltersUpdated?: (filters: Filter[]) => void;
+  buttonProps?: Partial<EuiButtonIconProps>;
+}
+
+export const AddFilterPopover = React.memo(function AddFilterPopover({
+  indexPatterns,
+  filters,
+  timeRangeForSuggestionsOverride,
+  onFiltersUpdated,
+  buttonProps,
+}: AddFilterPopoverProps) {
+  const [isAddFilterPopoverOpen, setIsAddFilterPopoverOpen] = useState(false);
+
+  const buttonIconLabel = i18n.translate('unifiedSearch.filter.filterBar.addFilterButtonLabel', {
+    defaultMessage: 'Add filter',
+  });
+
+  const button = (
+    <EuiToolTip delay="long" content={buttonIconLabel}>
+      <EuiButtonIcon
```

Comment:
This is what I meant by tooltips:
<img width="153" alt="Screen Shot 2022-04-12 at 15 38 56 PM" src="https://user-images.githubusercontent.com/549577/163066837-15f3a630-01c5-4b54-837b-b79b50ef33a9.png">